fop/fom.c

```diff
@@ -135,6 +135,7 @@
 
 enum {
 	LOC_IDLE_NR = 1,
+	LOCM_CHECK_PERIOD_SEC = 10*60,
```

Comment:
[* question *] Now it will print the warning message every hour. Is it what you had in mind?

Reply:
I had 10 mins in mind. Why every hour?

Reply:
Sorry, you're right. 10 minutes.